docker/install.sh

```diff
@@ -1,18 +1,13 @@
 #!/usr/bin/env bash
 
 apt-get update
-apt-get install -y build-essential libatlas-base-dev libopencv-dev graphviz
-apt-get install -y python-setuptools python-pip
-apt-get install -y openjdk-7-jdk
-apt-get install -y nginx
-
 apt-get install -y software-properties-common
 add-apt-repository -y ppa:certbot/certbot
 apt-get update
-apt-get install -y python-certbot-nginx
-apt-get install protobuf-compiler libprotoc-dev
 
-pip install --upgrade pip
-pip install gunicorn
-pip install gevent
-pip install mxnet-model-server
+apt-get install -y build-essential libatlas-base-dev libopencv-dev graphviz python-dev \
+    openjdk-8-jdk nginx protobuf-compiler libprotoc-dev python-certbot-nginx curl
+
+cd /tmp && curl -O https://bootstrap.pypa.io/get-pip.py && python get-pip.py
```

Comment:
hmm. seems a bit unclean... why not something along the lines of:
```
$ apt-get install python-pip 
$ pip install --upgrade pip 
```

Reply:
apt-get repo may not have latest version of pip (usually doesn't). this is the best way to install pip.

Reply:
Got it. Cool.